packages/twenty-server/src/modules/calendar/commands/start-google-calendar-sync.cron.command.ts

```diff
@@ -0,0 +1,31 @@
+import { Inject } from '@nestjs/common';
+
+import { Command, CommandRunner } from 'nest-commander';
+
+import { MessageQueue } from 'src/engine/integrations/message-queue/message-queue.constants';
+import { MessageQueueService } from 'src/engine/integrations/message-queue/services/message-queue.service';
+import { GoogleCalendarSyncCronJob } from 'src/modules/calendar/jobs/crons/google-calendar-sync.cron-job';
+import { googleCalendarSyncCronPattern } from 'src/modules/calendar/jobs/crons/google-calendar-sync.cron.pattern';
+
+@Command({
+  name: 'google-calendar-sync:cron:start',
```

Comment:
There is start but no stop. I'm fine with that but let's use another name.
You can look at "GmailPartialSyncCronCommand", let's also follow this pattern
"cron:$MODULE:$COMMAND"